port/unix_include/omrportpg.h

```diff
@@ -120,7 +121,8 @@ typedef struct OMRPortPlatformGlobals {
 #endif
 
 #if defined(LINUX)
-#define PPG_cgroupLimitsEnabled (portLibrary->portGlobals->platformGlobals.cgroupLimitsEnabled)
+#define PPG_cgroupSubsystemsAvailable (portLibrary->portGlobals->platformGlobals.cgroupSubsystemsAvailable)
```

Comment:
The condition `PPG_cgroupEntryList` being non-null is used only to check if `PPG_cgroupSubsystemsAvailable` and `PPG_cgroupSubsystemsAvailable` have already been set. As such there is no relation between the three.